src/ts/components/Match.tsx

```diff
@@ -32,6 +38,14 @@ class Match<TMatch extends IMatch> extends Component<IMatchProps<TMatch>> {
                 matchId={matchId}
                 suggestions={suggestionsToRender}
               />
+              <div className="MatchWidget__feedbackLink">
+                <a 
+                  target="_blank"
+                  href={getFeedbackLink(feedbackInfo)}
+                >
+                  Something's not right? Tell us!
```

Comment:
`nit` I wonder if there is a nice way to display this message, possibly an icon?

Reply:
There's a suggested icon in the mockup, but also this exact text.  I haven't done the icon ... yet.